src/devices/controller/analog-button.h

```diff
@@ -0,0 +1,32 @@
+#include <Arduino.h>
+#include <median-filter.h>
+
+#include "Types.hpp"
+
+class AnalogButton {
+ public:
+  AnalogButton(int pin);
```

Comment:
Nit: Please add a comment specifying that this will define the pin as an input.

Reply:
It doesn't do that - it doesn't set `inputMode` on the pin at all. The default is `INPUT`.